imagedephi/gui.py

```diff
@@ -24,9 +24,16 @@ def _load_template(template_name: str) -> str | None:
 async def lifespan(app: FastAPI) -> AsyncGenerator[None, None]:
     # Reset server state on startup, to support unit testing
     shutdown_event.clear()
+    # TODO: Use ASGI lifespan state, once it's supported by Hypercorn
```

Comment:
I think this would be better as an issue so we can track it and resolve when we are able.

Reply:
Done: #113